### PR TITLE
[SP-107] Logout

### PR DESCRIPTION
**Pull request by megifalko**

### CODE REVIEW

File: src/containers/HeadContainer.vue [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
-const userType = Cookies.get('user-type')
-const loggedIn = Cookies.get('logged-in')
+import Cookies from "js-cookie";
+import { logout } from "../services/api";
+import { ref, watch } from "vue";
+import { useRouter, useRoute } from 'vue-router'
+
+const router = useRouter();
+const route = useRoute();
+const userType = ref(Cookies.get('user-type'));
```
Comment (RazorMeister) [inline src/containers/HeadContainer.vue:21]: Use `useUserSession` service with `clearUserSession` function instead

**Review by RazorMeister — CHANGES_REQUESTED** [review]

File: src/containers/HeadContainer.vue (hunk 1 above)
Comment (megifalko) [reply src/containers/HeadContainer.vue:21]: `function logout() {
  clearUserSession();
  userSession.token = "";
  userSession.userType = "";
  router.push({ name: 'login' });
}`

Do I have to clear token and userType llike that?

Comment (RazorMeister) [reply src/containers/HeadContainer.vue:21]: Yes but I'd move it directly to the clearUserSession() function so this logout function would look like:
`function logout() { clearUserSession(); router.push({ name: 'login' }); }`

Comment (megifalko) [reply src/containers/HeadContainer.vue:21]: I inserted cleaning user token and type in the useUserSession file.

**Review by RazorMeister — APPROVED** [review]

**Review by dvalim — APPROVED** [review]

**Pull request merged**